catalog/spatial/registry/registry-admin-modules/registry-admin-local-ui/src/main/webapp/js/view/Association.view.js

```diff
@@ -38,8 +38,21 @@ define([
             events: {
                 "click .remove-association": 'removeAssociation'
             },
+            initialize: function(options) {
+                this.readOnly = options.readOnly;
```

Comment:
I'm assuming that all of the "read only" stuff that I'm seeing through the PR is meant only to change how the UI looks and not to actually enforce any sort of policy on editing?